src/shared/ui/widget-title/widget-title.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { WidgetTitle } from './widget-title';
+
+import styles from './widget-title.module.scss';
+
+describe('WidgetTitle component', () => {
+  it('renders without crashing', () => {
+    render(<WidgetTitle>TestTitle</WidgetTitle>);
+    const title = screen.getByText('TestTitle');
+
+    expect(title).toBeInTheDocument();
+  });
+
+  it('displays h2 tag', () => {
+    render(<WidgetTitle />);
+    const element = screen.getByRole('heading', { level: 2 });
+
+    expect(element).toBeInTheDocument();
+  });
+
+  it('displays size correctly when size=small', () => {
+    render(<WidgetTitle size="small">TestTitle</WidgetTitle>);
+    const element = screen.getByRole('heading', { level: 2 });
+
+    expect(element).toHaveClass(styles.small);
```

Comment:
Please, use classnames.bind()

Reply:
I've updated the tests. I can return to the old name, I’m worried about the confusion in imports. Which option should I choose?
1)cx()
2)widgetTitleClassNames()

Reply:
`cx()`